docs/source/build.md

```diff
@@ -57,9 +94,17 @@ In addition to the libraries included with cudatoolkit 11.0+, there are some oth
 - [Googlebench](https://github.com/google/benchmark) - Needed to build benchmarks
 - [Doxygen](https://github.com/doxygen/doxygen) - Needed to build docs
 
-All of RAFT's C++ APIs can be used header-only but pre-compiled shared libraries also contain some host-accessible APIs and template instantiations to accelerate compile times.
+#### Conda environment scripts
+
+Conda environment scripts are provided for installing the necessary dependencies to build both the C++ and Python libraries from source. It is preferred to use `mamba`, as it provides significant speedup over `conda`. In addition you will have to manually install a CUDA toolkit which is greater than, or equal to, the version you install into you conda environment. Installing CUDA toolkit into your host system is necessary because `nvcc` is not provided with Conda's cudatoolkit dependencies. The following example will install create and install dependencies for a CUDA 11.8 conda environment:
```

Comment:
This situation changes pretty dramatically (for the better) with CUDA 12. Should we figure out a way to tell the "simple" CUDA 12 story first, which has no system CTK requirement, and then describe the CUDA 11 case after that? (Eventually the CUDA 11 info will be safe to remove, so I'd try to separate it now.)